validation/validate_dc2_run2.1i_object_table.ipynb

```diff
@@ -224,6 +247,134 @@
     "print(f'Loaded {len(df)} objects from \"{catalog_name}\" with a subsampling of {sampling_factor}x')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### CosmoDC2\n",
+    "\n",
+    "Load CosmoDC2 for reference.  This is the truth table for the input catalog proposals.\n",
+    "It's larger than the Object Table, so we're going to use a similar subsampling plan."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Need to use 'galaxy_id' instead of 'objectId'\n",
+    "if subsample is None:\n",
+    "    cosmoDC2_subsample = None\n",
+    "else:\n",
+    "    cosmoDC2_subsample = (lambda x: x % sampling_factor == remainder, 'galaxy_id')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cosmoDC2_cat = GCRCatalogs.load_catalog('cosmoDC2', config_overwrite={'use_cache': False})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cosmoDC2_columns = ['ra', 'dec', 'mag_u', 'mag_g', 'mag_r', 'mag_i', 'mag_z', 'mag_y']\n",
+    "max_mag_i = 25\n",
+    "mag_limit_filter = [GCRQuery(f'mag_i < {max_mag_i}')]\n",
+    "cosmoDC2 = pd.DataFrame(cosmoDC2_cat.get_quantities(cosmoDC2_columns,\n",
+    "                                                    filters=[cosmoDC2_subsample] + mag_limit_filter))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f'Loaded {len(cosmoDC2)} objects from \"cosmocDC2\" with a subsampling of {sampling_factor}x')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### HSC XMM\n",
+    "from HSC PDR1."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hsc_cat = GCRCatalogs.load_catalog('hsc-pdr1-xmm',\n",
+    "                                   config_overwrite={'base_dir': '/global/cscratch1/sd/desc/hsc',\n",
+    "                                                    'use_cache': False})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Establish some basic cuts intended to yield a galaxy sample with reasonable flux measurements.\n",
+    "basic_cuts = [\n",
+    "    GCRQuery('clean'), # The source has no flagged pixels (interpolated, saturated, edge, clipped...) \n",
+    "                       # and was not skipped by the deblender\n",
+    "    GCRQuery('xy_flag == 0'),                                      # Flag for bad centroid measurement\n",
+    "]\n",
+    "\n",
+    "# Define cuts on galaxy properties: currently we simply make a sample limited at i<25.\n",
```

Comment:
I didn't see where any star/galaxy separation occurred. Is this really cuts on "object" properties?